exhibition/static/exhibition/css/exhibitors-public-detail.css

```
@@ -282,3 +282,147 @@
+    align-items: flex-end;
+}
+
+.exhibition-session-list .pretalx-session-info .track,
+.exhibition-session-list .pretalx-session-info .room {
+    flex: 1;
+    white-space: nowrap;
+    display: inline-block;
+    max-width: 100%;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    word-wrap: normal;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)